newsroom/templates/coverage_request_content.html

```diff
@@ -0,0 +1,7 @@
+{% block content %}
+<p>A coverage inquiry has been submitted by {{ name }} ({{ email }}) from {{ company }} regarding Agenda item: <a href="{{url}}">{{ url }}</a></p>
+
+Message:
+{% include "message.html" %}
+<a href="mailto:{{ email }}?cc={{ recipients|join(',') }}&subject={{ subject }}">Click to reply</a>
```

Comment:
There should probably be some spacing between the message and the `Click to reply` link

Reply:
There is a sufficient gap in the actual email because of the padding of message.